pkg/overlay/config.go

```diff
@@ -26,7 +27,8 @@ var (
 // Config is a configuration struct for everything you need to start the
 // Overlay cache responsibility.
 type Config struct {
-	DatabaseURL string `help:"the database connection string to use" default:"bolt://$CONFDIR/overlay.db"`
+	DatabaseURL     string `help:"the database connection string to use" default:"bolt://$CONFDIR/overlay.db"`
+	RefreshInterval int    `help:"the interval at which the cache refreshes itself in seconds" default: 30`
```

Comment:
oh nice.

one thing you can do is use a time.Duration instead of an int. the default can then be "30s" instead of just "30", or someone can provide "1h" or "30m". the config system knows how to handle time.Duration specifically

Reply:
Oh that's a neat little trick, I actually didn't know that. Will do!